agent/hcp/telemetry/otel_exporter.go

```diff
@@ -0,0 +1,66 @@
+package telemetry
```

Comment:
Let's add a doc.go with a package description. @clly  does this in the collector and it's quite nice.

Reply:
Done - would love feedback on the writeup.